facets_mcp/tools/ftf_tools.py

```
@@ -190,30 +268,32 @@ def validate_module(module_path: str, check_only: bool = False) -> str:
-        validation_results.append("Output Type Validation:")
-        validation_results.append("=" * 40)
+        run_ftf_command(command)
         
         # Use the utility function for output type validation
         success, validation_message = validate_module_output_types(module_path)
-        validation_results.append(validation_message)
-        
+        if not success:
+            return json.dumps({
+                "success": False,
+                "instructions": "Failed to validate module directory using FTF CLI. Try to fix the issues and run again, or ask the user to fix it if unclear what might be the issue.",
```

Pylint (3.3.7)

[convention] Line too long (184/100)

(C0301)